invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/globals/site.overrides

```diff
@@ -167,14 +167,13 @@ pre.export.result {
 }
 
 .featured-list {
-  font-weight: 200;
-  font-size: 4em;
+  font-weight: 500;
+  font-size: 6em;
   color: white;
   background-color: @featuredListBackgroundColor;
   box-sizing: border-box;
-  width: 100px;
-  height: 100px;
-  border-radius: 50%;
+  width: 100%;
+  height: 150px;
```

Comment:
do we need variables for easier customisation? Also font attributes above

Reply:
I don't think anyone will override this part. It is only used in the intro-section.

Reply:
in this case maybe it should have specific class name, otherwise I understand it is referring to featured comms or records, and we will override these clasess (discussed via chat)